dashboard/admin.py

```diff
@@ -79,5 +87,44 @@ def save_model(self, request, obj, form, change):
         obj.id = canvas_id_to_incremented_id(obj.canvas_id)
         return super(CourseAdmin, self).save_model(request, obj, form, change)
 
+class LogResource(resources.ModelResource):
+    class Meta:
+        model = Log
+
+# This is a local class for logs that adds in Export and disables some actions on the admin
+class MyLALogAdmin(ExportMixin, LogAdmin):
+    resource_class = LogResource
+    # Remove adding and editing for logs
+    @staticmethod
+    def has_add_permission(request):
+        return False
+    @staticmethod
```

Comment:
I would have thought this was flagged by Codacy, since I believe PEP8 has spaces between methods. Maybe we're not running PEP8 checkers there, IDK. Not a huge deal.

Reply:
I can fix that. It looks like PyLint doesn't do this check of 

E301 expected 1 blank line, found 0

https://stackoverflow.com/questions/6879640/how-can-i-configure-pylint-to-check-all-things-pep8-checks

Reply:
Okay fixed this